docs/_docs/storage/persistent.adoc

```diff
@@ -0,0 +1,90 @@
+// Licensed to the Apache Software Foundation (ASF) under one or more
+// contributor license agreements.  See the NOTICE file distributed with
+// this work for additional information regarding copyright ownership.
+// The ASF licenses this file to You under the Apache License, Version 2.0
+// (the "License"); you may not use this file except in compliance with
+// the License.  You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+= Persistent Storage
+
+//:javaFile: {javaCodeDir}/IgnitePersistence.java
+
+== Overview
+
+Ignite Persistence is designed to provide a quick and responsive persistent storage.
+When it is enabled for the link:config/data-region[data region], Ignite stores all the data on disk, and loads as much data as it can into RAM for processing.
```

Comment:
>When it is enabled for the link:config/data-region[data region]

Right now this sentence is incorrect, in Ignite 2 we had "persistent" flag in region, now we don't have it. Region is an attribute of the engine, it's persistent if the engine is persistent.

Reply:
Moreover, a data region is an attribute of some specific storage. Some storage can have data region, another can't. So all properties of persistence are storage specific.